src/components/StakingChart/index.js

```diff
@@ -1,46 +1,101 @@
-// import React from 'react';
-// import { AreaChart, Area, XAxis, CartesianGrid, Tooltip } from 'recharts';
-//
-// const data = [
-//   {
-//     name: 'Dec 1',
-//     uv: 2000,
-//   },
-//   {
-//     name: 'Dec 2',
-//     uv: 2500,
-//   },
-//   {
-//     name: 'Dec 3',
-//     uv: 2000,
-//   },
-//   {
-//     name: 'Dec 4',
-//     uv: 2400,
-//   },
-//   {
-//     name: 'Dec 5',
-//     uv: 1890,
-//   },
-// ];
-//
-// export default function StakingChart() {
-//   return (
-//     <AreaChart
-//       width={550}
-//       height={230}
-//       data={data}
-//       margin={{
-//         top: 10,
-//         right: 30,
-//         left: 0,
-//         bottom: 0,
-//       }}
-//     >
-//       <CartesianGrid strokeDasharray="3 3" />
-//       <XAxis dataKey="name" />
-//       <Tooltip />
-//       <Area type="monotone" dataKey="uv" stroke="#8884d8" fill="#8884d8" />
-//     </AreaChart>
-//   );
-// }
+import React from 'react';
+import { AreaChart, Area, XAxis, Tooltip } from 'recharts';
+
+const data = [
+  {
+    name: 'Dec 11',
+    uv: 1500,
+  },
```

Comment:
Should "Dec 11" be hardcoded here?

Reply:
I'm call Vitalii tomorrow and ask for this data